.devcontainer/devcontainer.json

```diff
@@ -1,15 +1,19 @@
 {
-    "name": "Salesforce Development Container",
-    "image": "salesforce/salesforcedx:latest-slim",
-    "extensions": [
-      "salesforce.salesforcedx-vscode",
-      "redhat.vscode-xml",
-      "dbaeumer.vscode-eslint",
-      "esbenp.prettier-vscode",
-      "salesforce.codey-midnight"
-    ],
-    "settings": {
-      "salesforcedx-vscode-apex.java.home": "/usr/lib/jvm/java-11-openjdk-amd64",
-      "workbench.colorTheme": "Codey Midnight"
+  "name": "Salesforce Development Container",
+  "image": "salesforce/salesforcedx:latest-slim",
+  "extensions": [
+    "salesforce.salesforcedx-vscode",
+    "redhat.vscode-xml",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "salesforce.codey-midnight",
+    "/root/workspace/sfdx-env/.devcontainer/code-builder-vscode-0.0.4.vsix"
+  ],
+  "settings": {
+    "salesforcedx-vscode-apex.java.home": "/usr/lib/jvm/java-11-openjdk-amd64",
+    "workbench.colorTheme": "Codey Midnight",
+    "files.exclude": {
+      "**/.vsix": true
```

Comment:
The plan was to hide the `welcome.sfdx` file as well however activation events in an extension do not work with hidden files.